docker/build-myst/Dockerfile

```diff
@@ -1,24 +1,18 @@
 FROM ubuntu:18.04
 
-ADD . /src
-WORKDIR /src 
-
 COPY ./build-myst.sh /build-myst.sh
 RUN /bin/chmod 777 /build-myst.sh
 
-RUN apt-get update
-RUN apt-get -y install git
+RUN apt-get update && \
+    apt-get -y install git sudo wget
```

Comment:
Why do we need `sudo`?

Reply:
We have `sudo` in a couple of places in our Makefiles.

During clean targets -
https://github.com/deislabs/mystikos/blob/main/rules.mak#L59
https://github.com/deislabs/mystikos/blob/main/Makefile#L88

prereqs installation - 
https://github.com/deislabs/mystikos/blob/main/prereqs/Makefile#L9-L24

Reply:
The prereqs installation could potentially be moved within the dockerfile.